### PR TITLE
Multi-select answertype

### PR DESCRIPTION
**Pull request by PeterSmallenbroek**

### CODE REVIEW

File: interfaces/HO-Portal/src/app/program/edit-person-affected-popup/edit-person-affected-popup.component.ts [hunk 1]
```diff
@@ -163,7 +163,10 @@ export class EditPersonAffectedPopupComponent implements OnInit {
           this.person.paTableAttributes[paTableAttribute.name].value;
 
         let options = null;
-        if (paTableAttribute.type === 'dropdown') {
+        if (
+          paTableAttribute.type === 'dropdown' ||
+          paTableAttribute.type === 'multi-select'
```
Comment (elwinschmitz) [inline interfaces/HO-Portal/src/app/program/edit-person-affected-popup/edit-person-affected-popup.component.ts:168]: These strings should be using the values from (one of) the Enum(s).

File: interfaces/HO-Portal/src/app/components/update-property-item/update-property-item.component.html [hunk 2]
```diff
@@ -70,6 +70,30 @@
                 class="ion-margin-top"
               ></ion-checkbox>
             </ng-container>
+            <ng-container *ngSwitchCase="'multi-select'">
```
Comment (elwinschmitz) [inline interfaces/HO-Portal/src/app/components/update-property-item/update-property-item.component.html:73]: These cases should also use the enum, similar to how the AW/PA templates do it.

File: interfaces/HO-Portal/src/app/models/fsp.model.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export enum AnswerType {
   Email = 'email',
   Boolean = 'boolean',
   String = 'string',
+  Dropdown = 'dropdown',
```
Comment (elwinschmitz) [inline interfaces/HO-Portal/src/app/models/fsp.model.ts:23]: There is already one for `"dropdown"`, see line 18: `Enum`.

**Comment by PeterSmallenbroek** [comment]
Closed as #3153 contains this

**Pull request closed without merging**